### ISSUE TITLE
Missing epel repository setup

### ISSUE THREAD

**Issue by nterray** [Erwyn/docker-tuleap#3]
When trying to install tuleap, I got the following error:

```
Error: Package: tuleap-6.11.99.42-1.noarch (Tuleap)
           Requires: ckeditor >= 4.3.2
```

As stated in the installation guide, epel repositories must be configured:
https://tuleap.net/wiki/index.php?pagename=Installation%20%26%20Administration%2FHow%20to%20install&group_id=101#Install_Epel

**Comment by Erwyn** [comment]
Lulz. One day it may work !

**Comment by Erwyn** [comment]
Won't fix
